tests/test_styles.py

```diff
@@ -1,15 +1,21 @@
 from . import unittest
 from shapely.geometry import CAP_STYLE, JOIN_STYLE
+from shapely.constructive import BufferCapStyles, BufferJoinStyles
 
 
 class StylesTest(unittest.TestCase):
-
     def test_cap(self):
```

Comment:
the tests below can be improved to simplify (e.g.):
```diff
-        self.assertEqual(CAP_STYLE.round, 1)
+        assert CAP_STYLE.round == 1
```
(the `self.assert*` functions were used with unittest, prior to pytest)

Reply:
updated to use `assert`, also moved the tests into `tests/test_buffer.py`